eq-author/src/components/IntroEditor/index.js

```diff
@@ -31,11 +31,15 @@ const AddIntroButton = styled(Button)`
   border: 1px solid ${colors.bordersLight};
 `;
 
-const IntroToolbar = styled(Toolbar)`
+const IntroToolbar = styled.div`
   border: 1px solid ${colors.bordersLight};
   border-bottom: 0;
-  background-color: ${colors.white};
   border-radius: ${radius} ${radius} 0 0;
+  padding: 1.5em;
+`;
+
+const StyledWrapper = styled.div`
```

Comment:
You don't need to create this additional component, just fix the one above—it's not really doing anything.

```
const IntroToolbar = styled.div`
  display: flex;
  padding: 1em 1em 0 1em;
`;
```